docs/integration/ibm-cloud-kubernetes-logs.md

```diff
@@ -0,0 +1,253 @@
+title: IBM Cloud Kubernetes Logs Integration
+description: Sematext IBM Cloud Kubernetes Logs integration is configured by running Logagent as a Daemonset in your cluster.
+
+With this integration you can:
+
+- Forward all Containerd container logs
+- Use log globs to choose which containers to log
+- Drop noisy logs with `dropEvents`
+- Forward logs to different Apps
+- Enable Kubernetes audit logs 
+
+IBM Cloud Kubernetes uses Cointainerd as the container engine. In this case Logagent can't use the Docker remote API to retrieve logs and metadata. Instead, logs are collected from containerd log files, which requires access to the relevant directories.
+
+The Logagent [input-filter for Containerd](../logagent/input-filter-containerd/) supports:
+
+* Tailing log files from `/var/log/containers/`, `/var/log/pods` and `/var/data/kubeletlogs`
+* Enrichment of logs with podName, namespace, containerName, containerId
+* Joining long log lines over 4KB into one log line
```

Comment:
Are they really lines, or is "events" a better word choice here?  Not sure, asking.

Reply:
Changed to "event".